kernel/ramfs.c

```diff
@@ -2292,7 +2292,9 @@ static int _fs_get_events(myst_fs_t* fs, myst_file_t* file)
     if (!_ramfs_valid(ramfs) || !_file_valid(file))
         ERAISE(-EINVAL);
 
-    ret = -ENOTSUP;
+    /* Regular files always poll TRUE for reads and writes */
```

Comment:
@mikbras does it make sense to add this to ext2 as well?

According to https://man7.org/linux/man-pages/man3/poll.3p.html -

>   Regular files shall always poll TRUE for reading and writing.

Reply:
Based on the pipeline run, it seems the changes are at least harmless. https://openenclave.visualstudio.com/ACC-Services/_build/results?buildId=9302&view=logs&j=ab477ad6-9781-5b54-82ca-a0141b26726e&t=2d9b6f4f-266f-5cad-a875-41dc232b2771